app/api/v1/views.py

```diff
@@ -88,35 +79,42 @@ def get_orders():
                                 multiple_items,
                                 start,
                                 size,
-                                "search")
+                                'search')
```

Comment:
Should we add search as a variable that can change here or not include it since we are using the default?

Reply:
Removed 'search' since we are using default.